docs/changelog.md

```diff
@@ -0,0 +1,12 @@
+# Changes from v6.0 to v7.0

```

Comment:
We already have a change log in the root directory, so probably don't need an additional one. Also, as this is the first public release, the changes being referred to below are already going to be included. This file can probably be deleted.

Reply:
done!